### PR TITLE
add post processing scripts

### PR DESCRIPTION
**Pull request by akshaysripada**
I have added the two post processing scripts for cleaning through gauge errors for all the scenarios in the batch and then create useful graphs for analysis.

### CODE REVIEW

**Comment by mandli** [comment]
A couple of comments:
 - Can you encapsulate the script's functionality all into functions and then provide the `if __name___ == "__main__":` business to run from the command line?
 - Provide some basic documentation for the functions or at the very least what overall the scripts are doing.

**Comment by akshaysripada** [comment]
I addressed both the comments you mentioned above in the two most recent commits.

File: batch/gauge_error_analysis.py [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+"""
+# ============================================================================
+#      Copyright (C) 2013 Kyle Mandli <kyle@ices.utexas.edu>
+#
+#          Distributed under the terms of the MIT license
+#                http://www.opensource.org/licenses/
+#
+# (post process scripts by Akshay Sriapda <as4928@columbia.edu>, 2017)
+# ============================================================================
+
```
Comment (mandli) [inline batch/gauge_error_analysis.py:12]: You actually can omit this in lieu of the license document in the root directory.  Feel free to update that as well with your name.

File: batch/post_process_data_analysis.py [hunk 2]
```diff
@@ -0,0 +1,624 @@
...
+"""
+# ============================================================================
+#      Copyright (C) 2013 Kyle Mandli <kyle@ices.utexas.edu>
+#
+#          Distributed under the terms of the MIT license
+#                http://www.opensource.org/licenses/
+#
+# (post process scripts by Akshay Sriapda <as4928@columbia.edu>, 2017)
+# ============================================================================
+
```
Comment (mandli) [inline batch/post_process_data_analysis.py:12]: Same goes here.  Probably should remove this from the other source files (old standard).

**Pull request merged**